lib/SILOptimizer/LoopTransforms/LICM.cpp

```diff
@@ -534,9 +534,21 @@ static bool handledEndAccesses(BeginAccessInst *BI, SILLoop *Loop) {
   return true;
 }
 
+static bool isCoveredByScop(BeginAccessInst *BI, DominanceInfo *DT,
```

Comment:
Is "scop" a typo or a term I don't know?

Reply:
Whoops! typo :) should be scope of course - thanks!